ui/src/components/app-menu/QasAppMenu.vue

```diff
@@ -76,6 +80,11 @@ export default {
       return !!(children || []).length
     },
 
+    shouldExpand ({ children, to }) {
+      return children?.length &&
```

Comment:
`return children?.length && this.$route.matched.some(item => item.path === to.path)` não cabem na msm linha? o limite de col é 120 se não couber faz assim

```js
children?.length && this.$route.matched.some(item => {
  return item.path === to.path
})
```